tests/full_wlm/test_generic_orc_launch_batch.py

```diff
@@ -83,19 +87,22 @@ def test_launch_cluster_orc_batch_single(fileutils, wlmutils):
     # batch = False to launch on existing allocation
     network_interface = wlmutils.get_test_interface()
     orc = exp.create_database(
-        6780, db_nodes=3, batch=True, interface=network_interface, single_cmd=True
+        wlmutils.get_test_port(), db_nodes=3, batch=True, interface=network_interface, single_cmd=True
     )
+
+    test_account = wlmutils.get_test_account()
+    if test_account:
+        orc.batch_settings.set_account(test_account)
     if wlmutils.get_test_launcher() == "lsf":
-        orc.batch_settings.set_account(wlmutils.get_test_account())
         orc.batch_settings.set_walltime("00:02")
+    else:
+        orc.batch_settings.set_walltime("00:02:00")
```

Comment:
All of a sudden, you reminded me that we can actually specify time like that in LSF, because we convert it to `hh:mm` format on the fly. Removing outdated lines.